wlauto/instrumentation/servo_power_monitors/__init__.py

```diff
@@ -117,6 +117,10 @@ def initialize(self, context):
         self.poller = None
         self.data = None
         self.stopped = True
+
+        if self.device.platform != "chromeos":
+            raise DeviceError("servo_power instrument only supports chromeOS devices.")
```

Comment:
Also this should be raising an `IntrumentError` rather than a `DeviceError` (the latter is reserved for when an error occurs on/with the device).